src/openapi_parser/specification.py

```diff
@@ -155,8 +155,8 @@ class Parameter:
     schema: Schema
     required: Optional[bool] = field(default=False)
     description: Optional[str] = None
-    # example: Optional[Any]  # TODO
-    # examples: list[Any] = field(default_factory=list)  # TODO
+    example: Optional[Any] = None
+    examples: list[Any] = field(default_factory=list)
```

Comment:
After checking the docs for v3 spec, it looks like it won't work with a list type. It should work with dict-like structure, where the `key` is a string and the `value` is an [Example object](https://swagger.io/specification/v3/#example-object), here you can find more details - [Parameter object](https://swagger.io/specification/v3/#parameter-object).

Reply:
Oh, I see. Thanks. People are amusing on changing spec every time :)